aspnetcore/mvc/controllers/filters.md

```diff
@@ -381,18 +381,20 @@ Because these filters implement <xref:Microsoft.AspNetCore.Mvc.Filters.IResultFi
 * An authorization filter that prevents execution of the action.
 * An exception filter that handles an exception.
 
-For example, the following filter always runs and returns a <xref:Microsoft.AspNetCore.Mvc.NotFoundResult> if a `null` <xref:Microsoft.AspNetCore.Mvc.ObjectResult> is returned from anywhere in the request processing pipeline:
+For example, the following filter always runs and sets an action result (<xref:Microsoft.AspNetCore.Mvc.ObjectResult>) with a *422 Unprocessable Entity* status code when content negotiation fails:
 
 ```csharp
-public class NotFoundAlwaysRunFilterAttribute : Attribute, 
-    IAlwaysRunResultFilter
+public class UnprocessableResultFilter : Attribute, IAlwaysRunResultFilter
 {
     public void OnResultExecuting(ResultExecutingContext context)
     {
-        if (context.Result is ObjectResult objResult && 
-            objResult.Value == null)
+        if (context.Result is StatusCodeResult statusCodeResult &&
+            statusCodeResult.StatusCode == 415)
         {
-            context.Result = new NotFoundResult();
+            context.Result = new ObjectResult("Can't process this!")
```

Comment:
Maybe a different message: 😄

Reply:
Not bad ...... if it were up to me, it would be even whacker ...

```csharp
new ObjectResult("Are you nuts?! I'm not processing this crap!")
```

Reply:
Seriously tho ... we need to keep that short b/c of the width limit on our code blocks. This is nice and **_narrow_** for the doc.

Reply:
Sure